HCore-Pusher/HCore-Pusher.csproj

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="PusherServer" Version="4.6.1" />
-	<PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.2.32">
+    <PackageReference Include="PusherServer" Version="5.0.0" />
```

Comment:
Major version change. Tested?

Reply:
I've created a fake `UserNotifications` and _pusherMessenger.SendMessagesAsync executed successfully